src/textual/css/_styles_builder.py

```diff
@@ -783,6 +785,48 @@ def process_scrollbar_gutter(self, name: str, tokens: list[Token]) -> None:
         else:
             self.styles._rules[name.replace("-", "_")] = value
 
+    def process_scrollbar_size(self, name: str, tokens: list[Token]) -> None:
+        def scrollbar_size_error(name: str, token: Token) -> None:
+            self.error(name, token, scrollbar_size_property_help_text(context="css"))
+
+        if not tokens:
+            return
+        if len(tokens) != 2:
+            scrollbar_size_error(name, tokens[0])
+        else:
+            token1, token2 = tokens
+
+            if token1.name != "number" or not token1.value.isdigit():
+                scrollbar_size_error(name, token1)
+            if token2.name != "number" or not token2.value.isdigit():
+                scrollbar_size_error(name, token2)
+
+            horizontal, vertical = int(token1.value), int(token2.value)
```

Comment:
I followed the convention we've been using so far for this kind of CS properties, with the 1st number being `<horizontal>` and the second one the `<vertical>`.

A funny aspect of this though is that they both actually impact the layout of the "other" axis, which was counter intuitive for me while I was writing the test for it :smile: 
--> setting a _horizontal_ scrollbar size of `3` will reduce the _vertical_ available space, and vice-versa

Reply:
Yes, that's been a constant source of pain when working with layouts!

BTW, this construct is slower than it needs to be.

When you do the following:

`a, b = foo, bar`

You are creating a tuple (foo, bar) then unpacking it. You can avoid that by doing the following:

```
a = foo
b = bar
```

Reply:
Alright, I'll replace the unpacking expression by a proper multi-statements one :slightly_smiling_face: